rcl/src/rcl/time.c

```diff
@@ -23,18 +23,13 @@
 #include "rcl/error_handling.h"
 #include "rcutils/time.h"
 
-// Process default ROS time sources
-static rcl_time_source_t * rcl_default_ros_time_source;
-static rcl_time_source_t * rcl_default_steady_time_source;
-static rcl_time_source_t * rcl_default_system_time_source;
 
 // Internal storage for RCL_ROS_TIME implementation
-typedef struct rcl_ros_time_source_storage_t
+typedef struct rcl_ros_clock_storage_t
```

Comment:
Would `rcl_clock_storage_t` make more sense? Maybe not, not sure.

Reply:
This is specific to the ros subtype. It's not used by the system or steady time types.